### ISSUE TITLE
[a11y-defect-2]: Incorrect heading levels on process list component

### ISSUE THREAD

**Issue by coforma-jamie** [department-of-veterans-affairs/va.gov-team#64130]
### Point of contact

Jamie Klenetsky Fay

### Severity level

2, Serious. Should be fixed in 1-2 sprints post-launch.

### Details

On the process list component on the "Introduction" page, "Prepare" is an `<H3>`, and all other headings are an `<H4>`.

![image](https://github.com/department-of-veterans-affairs/va.gov-team/assets/135633989/525cb128-17af-499e-a857-e2956e0c28db)

This is the incorrect hierarchy. Someone navigating by heading would assume that "When you apply..." "Apply," "VA review," and "Decision" are all contained within the "Prepare" section.

### Reproduction steps

1. Go to [this page](https://staging.va.gov/careers-employment/vocational-rehabilitation/apply-vre-form-28-1900/start)
2. Click "Apply for Veteran Readiness and Employment with VA Form 28-1900"
3. Scroll down to the process list component (the "Prepare" heading):
![image](https://github.com/department-of-veterans-affairs/va.gov-team/assets/135633989/21ad9688-09cf-411a-b3e8-42f88f3321bc)

### Proposed solution or next steps

Change the heading levels: "Apply," "VA review," and "Decision" should all be `<H3>`.

### References, articles, or WCAG support

1. [Process list component](https://design.va.gov/components/process-list)
2. [W3.org - Heading ranks](https://www.w3.org/WAI/tutorials/page-structure/headings/#heading-ranks)

### Type of issue

- [X] Screenreader
- [X] Keyboard
- [ ] Focus
- [X] Headings
- [ ] Color
- [ ] Zoom
- [ ] Semantics
- [ ] Axe-core
- [ ] Something else

**Comment by zack** [comment]
Found the same problem in 646, so fixing that too.
